sdk/ai/azure-ai-inference/azure/ai/inference/_configuration.py

```diff
@@ -82,8 +82,8 @@ class EmbeddingsClientConfiguration:  # pylint: disable=too-many-instance-attrib
 
     :param endpoint: Service host. Required.
     :type endpoint: str
-    :param credential: Credential used to authenticate requests to the service. Is one of the
-     following types: AzureKeyCredential, AzureKeyCredential, TokenCredential Required.
+    :param credential: Credential used to authenticate requests to the service. Is either a key
+     credential type or a token credential type. Required.
```

Comment:
This change is expected, known tool change that came in. It came up in previous discussion with Azure SDK folks.